recipes/mbits-args/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.12)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
```

Comment:
```suggestion
 # FIXME: This project installs without a namespace https://github.com/mbits-libs/args/blob/72f5f2b87ae39f26638a585fa4ad0b96b4152ae6/CMakeLists.txt#L152
target_link_libraries(example ${CONAN_LIBS})
```

Reply:
Same here, does the 9c093cd0d1b4f8b2385948a472dd381fce1853a5 fix this issue?

Reply:
I am sorry, I was not very clear... There's a feature missing from conan-io/conan#7615 that's required to match the upstream behavior

What you had originally was sufficient, but it would be appreciated if you can add the comment above so we can trace back all the ones which will need to be fixed in the future

Reply:
Ok, got it now, added the comments